packages/graph/hash_graph/bench/benches/util.rs

```diff
@@ -0,0 +1,195 @@
+use std::{mem::ManuallyDrop, str::FromStr};
+
+use graph::{
+    ontology::AccountId,
+    store::{
+        AsClient, DataTypeStore, DatabaseConnectionInfo, DatabaseType, EntityTypeStore,
+        LinkTypeStore, PostgresStorePool, PropertyTypeStore, StorePool,
+    },
+};
+use tokio::runtime::Runtime;
+use tokio_postgres::NoTls;
+use type_system::{DataType, EntityType, LinkType, PropertyType};
+
+type Pool = PostgresStorePool<NoTls>;
+pub type Store = <Pool as StorePool>::Store<'static>;
+
+// TODO - deduplicate with integration/postgres/mod.rs
+pub struct StoreWrapper {
+    bench_db_name: String,
+    source_db_pool: Pool,
+    _pool: ManuallyDrop<Pool>,
+    pub store: ManuallyDrop<Store>,
+}
+
+impl StoreWrapper {
+    pub async fn new(bench_db_name: &str) -> Self {
+        let source_db_connection_info = DatabaseConnectionInfo::new(
+            // TODO - get these from env
```

Comment:
This was also mentioned in the PR description, will create and attach an asana task if people agree with it

Reply:
This has been done